samples/language/DemoProject/qsharp.json

```diff
@@ -0,0 +1,14 @@
+{
+    "dependencies": {
+        "Foo": {
+            "path": "../DependencyA"
+        },
+        "GitHub": {
+            "github": {
+                "owner": "sezna",
+                "repo": "qsharp-project-template",
+                "ref": "b4575725349f01183c98858c6ce8342cf490c436"
```

Comment:
do we want to check this in with a dependency to a personal repo? Would it make sense to have it in this repo, or does that get weird?